### DISCUSSION TITLE
Hero movement button changes to active on objects which don't have any interaction

### DISCUSSION THREAD

**Discussion by drevoborod** [ihhub/fheroes2#8136] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Linux

### Describe the bug

![изображение](https://github.com/ihhub/fheroes2/assets/10282847/3e2962f1-e249-4418-8949-59f4346797c3)
The "Hero movement" button is in "interactive" form, but nothing happens if entering already possesed mine, so it's useless to make the button to look like that in this case. Possibly there are some other objects which are technically interactive, but interaction has no effect on them. And I know that button state reflects "interactive" state of the coursor, but for me it looks weird when I press button and nothing happens. Interactive coursor on possesed mines is OK for me because it just shows that the object is potentially interactive.

### Save file

No save file required.

**Comment by oleg-derevenetz** [comment]
> And I know that button state reflects "interactive" state of the coursor, but for me it looks weird when I press button and nothing happens. Interactive coursor on possesed mines is OK for me because it just shows that the object is potentially interactive.

So interactive cursor on no-op objects is OK, but interactive button is not? To be honest, it looks like a matter of personal taste, and not a bug.

**Comment by ihhub** [comment]
As @oleg-derevenetz explained such a change will introduce inconsistency in gameplay which we should avoid.

I am moving this topic to the discussion section.
